pyproject.toml

```
@@ -4,19 +4,43 @@ version = "0.1.0"
 description = "A collection of cloud function related to Blaise Case Management Application (CMA)"
 authors = ["James Anthony Williams <81305008+JAWilliamsONS@users.noreply.github.com>"]
```

Trivial change, but to align with the other Python repos, we could change the Authors to the following:
```suggestion
authors = ["Blaise Team"]
```

The current one is a former B5 dev.